Dockerfile

```diff
@@ -14,28 +14,28 @@ USER node
 RUN npm install --omit=dev
 
 
-FROM node:lts
+FROM node:lts-alpine3.16
```

Comment:
This has been held up so long that 2 alpine versions came out, so it should be 3.18 now :)